src/modules/profile/selectors.ts

```diff
@@ -1,7 +1,28 @@
 import { createSelector } from '@reduxjs/toolkit'
 import { LOAD_PROFILE_REQUEST } from 'decentraland-dapps/dist/modules/profile/actions'
-import { getLoading } from 'decentraland-dapps/dist/modules/profile/selectors'
+import { getLoading, getProfileOfAddress } from 'decentraland-dapps/dist/modules/profile/selectors'
+import { getAddress } from 'decentraland-dapps/dist/modules/wallet/selectors'
 
 export const isLoadingProfile = createSelector([getLoading, (_state, address) => address], (loadingState, address) =>
   loadingState.some(action => action.type === LOAD_PROFILE_REQUEST && action.payload.address === address.toLowerCase())
 )
+
+const getBlockedAddressesOfProfile = createSelector(
+  [(state, address?: string) => (address ? getProfileOfAddress(state, address) : null)],
+  profile => profile?.avatars[0]?.blocked?.map(address => address.toLowerCase()) ?? []
+)
+
+// TODO: should we check that the logged user is not blocked by themselves?
```

Comment:
@LautaroPetaccio @flobarreto WDYT?

Feel free to merge the following suggestion 😉 
```suggestion
```

Reply:
No need to!